### PR TITLE
Task timetamps and limits in metadata response

### PR DESCRIPTION
**Pull request by aaithal**
### Summary

Task timetamps and limits in metadata response

### Testing

- [x] Builds on Linux (`make release`)
- [x] Builds on Windows (`go build -out amazon-ecs-agent.exe ./agent`)
- [x] Unit tests on Linux (`make test`) pass
- [x] Unit tests on Windows (`go test -timeout=25s ./agent/...`) pass
- [x] Integration tests on Linux (`make run-integ-tests`) pass
- [x] Integration tests on Windows (`.\scripts\run-integ-tests.ps1`) pass
- [x] Functional tests on Linux (`make run-functional-tests`) pass
- [x] Functional tests on Windows (`.\scripts\run-functional-tests.ps1`) pass

New tests cover the changes: Yes

### Licensing

This contribution is under the terms of the Apache 2.0 License: Yes

### CODE REVIEW

File: agent/app/agent_unspecified.go [hunk 1]
```diff
@@ -20,8 +20,15 @@ import (
...
 	"github.com/aws/amazon-ecs-agent/agent/engine"
 	"github.com/aws/amazon-ecs-agent/agent/engine/dockerstate"
+	"github.com/cihub/seelog"
 )
 
 func (agent *ecsAgent) initializeTaskENIDependencies(state dockerstate.TaskEngineState, taskEngine engine.TaskEngine) (error, bool) {
 	return errors.New("unsupported platform"), true
 }
+
+// startWindowsService is not supported on Linux
```
Comment (jhaynes) [inline agent/app/agent_unspecified.go]: s/Linux/Unspecified Platforms/

File: agent/app/agent_unspecified.go [hunk 2]
```diff
@@ -20,8 +20,15 @@ import (
...
+func (agent *ecsAgent) startWindowsService() int {
+	seelog.Error("Windows Services are not supported on unspecified platform")
```
Comment (jhaynes) [inline agent/app/agent_unspecified.go]: s/platform/platforms/

File: agent/app/agent_unspecified.go [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// +build !linux
+// +build !linux,!windows
```
Comment (richardpen) [inline agent/app/agent_unspecified.go]: Not sure if we still need this file, as currently we only support windows and linux.

Comment (aaithal) [reply agent/app/agent_unspecified.go]: i develop on my mac. being able to build on all platforms is still a useful thing to adhere to

**Pull request merged**